### PR TITLE
resource/alicloud_db_instance: Restoring the usage of a three node RDS

### PR DESCRIPTION
**Pull request by chaitiansheng0524**

### CODE REVIEW

File: alicloud/resource_alicloud_db_instance.go [hunk 1]
```diff
@@ -411,7 +411,6 @@ func resourceAliCloudDBInstance() *schema.Resource {
 				Optional: true,
 				Computed: true,
 				ForceNew: true,
-				Removed:  "The parameter 'zone_id_slave_b' has been removed from provider version v1.213.1.",
```
Comment (MrWolong) [inline alicloud/resource_alicloud_db_instance.go:414]: 这个为什么去掉

File: website/docs/r/db_instance.html.markdown [hunk 2]
```diff
@@ -658,7 +658,7 @@ The following arguments are supported:
 
 -> **NOTE:** For more information about minor engine versions, see Release notes of minor AliPG versions, Release notes of minor AliSQL versions, and Release notes of minor engine versions of ApsaraDB RDS for SQL Server.
 * `zone_id_slave_a` - (Optional, ForceNew, Available since 1.101.0) The region ID of the secondary instance if you create a secondary instance. If you set this parameter to the same value as the ZoneId parameter, the instance is deployed in a single zone. Otherwise, the instance is deployed in multiple zones.
-* `zone_id_slave_b`- (Removed since v1.213.1) The parameter 'zone_id_slave_b' has been removed from provider version v1.213.1.
+* `zone_id_slave_b` - (Optional, ForceNew, Available since 1.230.1) RDS MySQL Cluster series instances support creating 1 to 2 secondary nodes at the same time when establishing a new instance. If you have this requirement, you can use this parameter to specify the availability zone for the second secondary node.
```
Comment (MrWolong) [inline website/docs/r/db_instance.html.markdown:661]: 1.231.1

Comment (chaitiansheng0524) [reply website/docs/r/db_instance.html.markdown:661]: ok

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**